parallel-lint

```diff
@@ -44,6 +44,13 @@ $autoloadLocations = array(
     __DIR__ . '/../../autoload.php',
 );
 
+if (class_exists('Phar') && Phar::running() !== '') {
```

Comment:
It makes sense. What about extract idea to variable name? Something like

```
$needSetAutoloadForPhar = class_exists('Phar') && Phar::running() !== '';
if ($needSetAutoloadForPhar) {
    $autoloadLocations = array(
        __DIR__ . '/vendor/autoload.php',
    );
}
```

Reply:
We have :) someone here with love for temporary variables.

Reply:
@grogy As the condition is not used anywhere else, assigning to a temporary variable does not make sense to me.